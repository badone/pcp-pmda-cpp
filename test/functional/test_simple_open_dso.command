TEMP_DIR=`mktemp -d -t test_simple_open_dso-XXXXXXXXXX`
./pmdasimple --export-all "$TEMP_DIR"
if [ $? -ne 0 ]; then exit; fi
dbpmda -e -n "$TEMP_DIR/root" <<EOF
open dso libpmda_simple.so simple_init
close
EOF
rm -rf "$TEMP_DIR"
