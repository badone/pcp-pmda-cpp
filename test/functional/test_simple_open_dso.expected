open dso libpmda_simple.so simple_init
close
